pypdns/__init__.py,sha256=nzVfNaGBZe-jNlNpokThsXAU5Y0yevIY3eGB1AZV2FU,118
pypdns/api.py,sha256=-k1hFVcsHE05zZc-o3IbuGdPEFIdtjQY4O4C-LHwhgg,3053
pypdns/errors.py,sha256=yB-CsXMP87gyqfvqQBXuOy7YhKOpxF7GYFn5_2ZQQKc,215
pypdns-1.4.1.dist-info/LICENSE,sha256=o1p6jKkqoo0k2KXZ23F8zTPaRGIoF8Gu9ADxi-QiUyo,1576
pypdns-1.4.1.dist-info/METADATA,sha256=ku5RoWMcZLRK4BNcCPG-FdaivsJYpzxMDNshL2If0Ww,1365
pypdns-1.4.1.dist-info/WHEEL,sha256=U88EhGIw8Sj2_phqajeu_EAi3RAo8-C6zV3REsWbWbs,92
pypdns-1.4.1.dist-info/top_level.txt,sha256=i73xSBgVDvUaV-6i6Fz997SwcXtjeeaxYAdcMLnT1jo,7
pypdns-1.4.1.dist-info/RECORD,,
